packages/ra-core/src/dataProvider/useGetManyAggregate.ts

```diff
@@ -66,15 +67,13 @@ export const useGetManyAggregate = <RecordType extends Record = Record>(
 ): UseGetManyHookValue<RecordType> => {
     const dataProvider = useDataProvider();
     const queryClient = useQueryClient();
+    const queryCache = queryClient.getQueryCache();
     const { ids } = params;
     const placeholderData = useMemo(() => {
-        const records = ids.map(id =>
-            queryClient.getQueryData<RecordType>([
```

Comment:
see the linter warning, you've added a new dependency to useMemo

Reply:
Amended. Will remember to lint in the future.